contrib/devtools/install-golangci-lint.sh

```diff
@@ -31,4 +31,4 @@ echo "Checking hashsum ..." >&2
 chmod +x "${installer}"
 
 echo "Launching installer ..." >&2
-exec "${installer}" -d -b "${GOBIN}" "${VERSION}"
\ No newline at end of file
+exec "${installer}" -d -b "${GOBIN}" "${VERSION}"
```

Comment:
Isn't this the same line?

Reply:
New line added (my code editor enforcing this now).
It already covered in other PR so I will revert this file